src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx

```diff
@@ -0,0 +1,62 @@
+export const ALL_VIEWS = [
+  {
+    id: 'all',
+    name: 'All Events',
+    data: {
```

Comment:
Do these fields mean we're going to be using discover queries?

Reply:
I don't expect we'll use Discover queries at all. I think this looks similar because we have a 1:1 mapping to a few Snuba fields. This format might be subject to change in future though.

Reply:
Could we use discover queries? If its possible it would let us have 1 fewer way to send queries from the front-end to the API.

Reply:
We can extract some utilities for common behavior, I don't think we can keep the format wholesale though since there will be quite a few differences  e.g. in search syntax, groupings

Reply:
Yeah I don't think we would want to use discover queries (esp because of search)